README.md

```diff
@@ -0,0 +1,109 @@
+# 🛸Project Goals🛸
+
+These are snatched from the original project (_OP_), with some adjustments and amendments.
+Even though the list is ordered, they are not (yet) ordered by priority, but merely to make each point adressable, as in `🛸2`.
+
+1. provide an idiomatic rust implementation for Google APIs, which includes _type safety_ and native _async_ operations.
```

Comment:
+1, but we may disagree on the path to async. I'm intending to write a synchronous version first, and once the rust ecosystem delivers async/await to move it over. I expect async/await to land in 1.38 or 1.39 and that hyper and reqwest will be pretty fast followers once it's landed on stable. I know hyper is already working towards it on the master branch. Once the ecosystem has a stable async/await it should be relatively easy to port the synchronous code into async code so I don't feel the need to jump through the hoops to deliver async before then.

Reply:
Staying true to what [I said here](https://github.com/Byron/github-star-counter), I agree that starting `sync` and going `async` will be pretty straightforward. Starting sync will make life easier, too, and as of now, everything related to async https is a pain I don't want to have (again).

Yes, let's start `sync`.